tests/neural_compressor/test_optimization.py

```diff
@@ -229,7 +237,7 @@ def compute_metrics(p: EvalPrediction):
                 transformers_outputs = transformers_model(**tokens)
             self.assertTrue(torch.allclose(onnx_outputs.logits, transformers_outputs.logits, atol=1e-4))
             sparsity = trainer.get_model_sparsity()
-            self.assertGreaterEqual(sparsity, target_sparsity * 100)
+            self.assertGreaterEqual(sparsity, target_sparsity * 100 / 2)
```

Comment:
Where does the  `/ 2` comes from ?

Reply:
"get_model_sparsity()" return the sparsity of the whole model, in the model, no all ops will be pruned, there are many op unpruned, and target_sparsity is only indicates that the sparsity for the op that can do prune. In this case, if we set the target sparsity to 0.9,  the model sparsity is about 0.56, So I divide the result by 2.

Reply:
@echarlaix, I add the explanation of pruning in the README and docs/source/optimization_inc.mdx. 
Thanks!

Reply:
Thank you @PenghuiCheng